sdk-commons/src/main/java/org/openmrs/maven/plugins/utility/NPMHelper.java

```diff
@@ -0,0 +1,43 @@
+package org.openmrs.maven.plugins.utility;
+
+import com.google.gson.JsonElement;
+import com.google.gson.JsonObject;
+import com.google.gson.JsonParser;
+import org.apache.http.client.methods.HttpGet;
+import org.apache.http.impl.client.CloseableHttpClient;
+import org.apache.http.impl.client.HttpClients;
+import org.apache.http.util.EntityUtils;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Comparator;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+public class NPMHelper {
```

Comment:
We've got [this coming in](https://github.com/openmrs/openmrs-sdk/pull/280/files#diff-ce36aadad63bc1048ab5ad01366f71c377394aae0a0d22d8181f2ca0db7f9cd4R1) as well. Which is, I think, a little more correct (i.e., it can support alternative registries, support for registries requiring authentication, etc.)

Reply:
@ibacher should I wait for it to get merged?

Reply:
Yes, I think so.